daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/EqualitySpec.scala

```diff
@@ -44,7 +44,8 @@ class EqualitySpec extends WordSpec with Matchers with TableDrivenPropertyChecks
   private val VariantCon1: Ref.Name = "Left"
   private val VariantCon2: Ref.Name = "Right"
 
-  private val struct2Fields = ImmArray[Ref.Name]("fst", "snd")
+  private val struct2Fields =
+    Struct.assertFromSeq(List[Ref.Name]("fst", "snd") zip Stream.continually(()))
```

Comment:
Why not like in all other places?
```suggestion
    Struct.assertFromSeq(List(n"fst" -> (()), n"snd" -> (())))
```